<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tree</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 227, 221);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(202, 195, 212);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(229,227,221);
	line-height:10px;
	font: 22pt &quot;Cascadia Mono&quot;;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>56</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>IQ-TREE</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="iqOutGroup">
                  <property name="text">
                   <string>OutGroup</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="iqGroupSelect">
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
}
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="iqBootstrap">
                  <property name="text">
                   <string>Bootstrap</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="bootSelect">
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
}
</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Ultrafast</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Standard</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>None</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="iqNumStrap">
                  <property name="text">
                   <string>Num of Bootstrap</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="iqNum">
                  <property name="styleSheet">
                   <string notr="true">    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
</string>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="text">
                   <string>1000</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Models</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="mod">
                  <property name="styleSheet">
                   <string notr="true">    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
</string>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="text">
                   <string>AUTO</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="iqtree">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
                </property>
                <property name="text">
                 <string>Generate</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pic">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
        </property>
        <property name="text">
         <string>View Pictures</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
